src/cljs/org/broadinstitute/firecloud_ui/utils.cljs

```diff
@@ -76,6 +76,11 @@
 
 (def access-token (atom nil))
 
+(defn get-cookie-domain []
+  (if (= "local.broadinstitute.org" js/window.location.hostname)
+    "local.broadinstitute.org"
+    (join "." (rest (split js/window.location.hostname ".")))))
+
```

Comment:
@dmohs - My PR doesn't make any changes to utils.cljs ... do you want this included in #420?